modules/common/certificates/helm/certificates/templates/rolebinding.yaml

```diff
@@ -11,9 +11,9 @@ metadata:
     "helm.sh/hook-delete-policy": "before-hook-creation,hook-succeeded,hook-failed"
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: {{ .Release.Name }}
+  kind: ClusterRole
+  name: lead-system-certificate-watch
 subjects:
 - kind: ServiceAccount
-  name: {{ .Release.Name }}
+  name: default
```

Comment:
i'd rather not use the `default` service account.  the pattern that we've used elsewhere is to create the sa, role and role binding in terraform, and then pass the sa to the helm chart as a value...can we go with that pattern for consistency and least privilege?